### DISCUSSION TITLE
How do I know if RetroAchievements on Miyoo Mini+ is working? If it's not working what do I do?

### DISCUSSION THREAD

**Discussion by windir19** [OnionUI/Onion#1114] [Q&A]
I don't know if anyone else has had my same struggles. I have been scouring reddit, the internet and even this Wiki for ways to make RetroAchievements work on my MiyooMini+ with Onion OS. It doesn't seem like there is a lot of information about how to specifically use RetroAchievements with the Miyoo Mini+. When I have used Retroarch on my PC or Steam Deck, a notification shows up showing the game I am playing. Even if I don't get an achievement, it shows on my RetroAchievements profile the last game I played with a timestamp of when I played. This is not happening with any games I play that are on the GB, GBC, GBA, SNES, PS1, etc. on my Miyoo Mini+

Here is what I have done so far:

- I installed what I believe is the latest firmware for the Miiyoo Mini+ (version 202305052130)
- I have Onion V4.2.0-RC installed
- I am able to connect to my home internet and went to another place to try their internet.
- I used to have something that said before games "Cannot connect to RetroAchievements" that was fixed once I updated the firmware.
- I have gone into Retroarch settings and turned on Achievements. I have entered in my log-in information and made sure to save the configuration settings to Retroarch.
- I turned on under settings for "Startup Summary" the setting to show "Games with Achievements"

Here my questions:

- How do I know when RetroAchievements should be working for me on the Miyoo Mini+? Would I see that I have played it on my RetroAchievements home page? Shouldn't I be getting a notification on the bottom telling me that it has achievements it can record?
- I know for SNES the Snes9x 2010 work with RetroAchievements but maybe the default core Beetle Supafaust doesn't work? In my testing I have been trying to use Snes9x 2010 Legend of Zelda Link to the Past and the defaults for other systems. Are there other cores that do work for each platform? (I mostly care about GB, GBC, GBA and SNES games)
- What is something else I should be considering that I haven't done above or something else that I overlooked as a limitation?
- Should I hold tight for an the actual release of Onion V4.2.0 will fix my issues with RetroAchievements?

Thank you for your time.

**Comment by dreams897** [comment]
If you input and save your Retroarch Achievements login info correctly into Retroarch it should display the number of achievements unlocked at the bottom of the screen when you launch the game, for example Achievements unlocked: 0 out of 250. I personally have not been able to get beyond that message when playing Pokemon Unbound, however and I am sure that I have unlocked some achievements. Additionally, I find that when WiFi and Retroarch Achievements are enabled, it slows things down a bit, which doesn't work for me personally.
